Tests/SourceKitLSPTests/RenameTests.swift

```diff
@@ -71,96 +75,110 @@ final class RenameTests: XCTestCase {
   }
 
   func testRenameFunctionBaseName() async throws {
-    try await assertRename(
+    try await assertSingleFileRename(
       """
       func 1️⃣foo() {}
-      foo()
+      2️⃣foo()
+      _ = 3️⃣foo
       """,
       newName: "bar()",
       expected: """
         func bar() {}
         bar()
+        _ = bar
         """
     )
   }
 
   func testRenameFunctionParameter() async throws {
-    try await assertRename(
+    try await assertSingleFileRename(
       """
       func 1️⃣foo(x: Int) {}
-      foo(x: 1)
+      2️⃣foo(x: 1)
+      _ = 3️⃣foo(x:)
```

Comment:
Could check the argument locations here too

Reply:
Renaming the argument labels currently isn’t supported. I filed rdar://119875277 to implement it in a follow-up PR because it might require some non-trivial refactoring.